lib/tasks/vacancies_to_update.yaml

```diff
@@ -212,3 +213,485 @@ We can offer:
+      <p>This post may involve working with children, vulnerable adults or dealing with sensitive information. Please note that in all cases written references will be taken up and made available to interviewers before the final selection stage; even if you indicate otherwise</p>"
+      experience: 'We are looking to appoint a colleague who will bring experience of supporting and raising achievement for vulnerable learners, including SEND, a vision, and leadership skills to aspire and drive whole school initiatives as we continue our journey to becoming Outstanding.'
+    -
+      slug: 'primary-semh-teacher'
+      job_title: 'Primary SEMH Teacher'
+      job_description: "<p>We are looking for a teacher to join our small team.</p>
+      <p>You must be a qualified teacher and preferably experienced.</p>
+      <p>You must have a genuine love of working with children, a desire to develop the whole child and an understanding of and belief in close multi-agency working.</p>
+      <p>You will be emotionally secure, resilient, have a great sense of humour and above all value the importance of working in a team. <br>
+      Closing date: noon on 2nd May 2018<br>
+      Interviews to be held on 9th May 2018
+      </p>
+      <p>Further details and application form can be downloaded from www.educationvillage.org.uk. Completed applications should be returned to dstapleton@educationvillage.org.uk.</p>
+      <p> For an informal discussion or to arrange a visit please contact Mandy Southwick on 01325 254670 or 07985 478799.</p>
+      <p><em>The Education Village Academy Trust is committed to safeguarding and promoting the welfare of young people.  This post is subject an enhanced DBS check and disqualification by association declaration.</em></p>"
+      benefits: 'Teacher Mainscale plus SEN 1 allowance'
+      experience: 'The successful candidate will be expected to demonstrate:
+      <ul><li>proven excellent classroom practice with the ability to engage and motivate all learners</li>
+      <li>knowledge of effective assessment practice to ensure progress is tracked and informs further development for each individual</li>
+      <li>clear commitment towards raising standards and expectations to enable all learners to reach their potential</li>
+      <li>excellent organisation skills with a commitment to professional development</li>
+      <li>a demonstrable record of outstanding classroom practice – commitment, dedication and enthusiasm in sharing our vision for the development of our community</li>
+      <li>a commitment to child protection procedures and policy and an understanding of safeguarding</li></ul>'
+      qualifications: 'Qualified Teacher Status'
+      starts_on: "01/09/2018"
+      ends_on: "31/08/2019"
+    -
+      slug: 'english-literacy-amp-enrichment-co-ordinator'
```

Comment:
When there are a subset of fields like this does it mean that the only update to do for this vacancy is the job_description?

Reply:
yes, this job specifically only needs the title updated.